ex3.js

```diff
@@ -0,0 +1,19 @@
+/*Scrieți o funcție pentru a găsi numărul de apariții al unei litere într-un șir
+Funcția letterCount() returnează numărul de apariții al unei litere trimise ca parametru dintr-un șir dat.
+*/
+
+const letterCount = (str, letter)=> {
+    str = str.toLowerCase();
+    let count = 0;
+        for (i=0; i < str.length; i++) {
```

Comment:
Acest for trebuie sa fie identat in acelasi rand cu `let count = 0;` de mai sus.
In caz contrar - se intelege gresit